docs/src/pages/component-demos/grid-list/AdvancedGridList.js

```diff
@@ -0,0 +1,71 @@
+// @flow weak
+
+import React from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import customPropTypes from 'material-ui/utils/customPropTypes';
```

Comment:
I have been documenting how to use the `withStyles()` Higher-order Component and migrating the whole demos to use it. I think that we should do it too. For reference:
- #6798
- #6810